src/scenic/simulators/carla/simulator.py

```diff
@@ -172,3 +174,13 @@ def getProperties(self, obj, properties):
 			angularSpeed=utils.carlaToScenicAngularSpeed(currAngVel),
 		)
 		return values
+
+	def destroy(self):
```

Comment:
Looks like this got accidentally pulled in from the other PR.

Reply:
Indeed, a `self.world.tick()` was added to that function, so it got pulled. It'd suggest waiting for PR #11 to be merged